.github/workflows/build-test.yml

```diff
@@ -273,7 +277,13 @@ jobs:
         with:
           crate: cargo-make
           version: latest
-
+      - name: Install emsdk
+        run: |
+          cd ~
+          git clone https://github.com/emscripten-core/emsdk.git
+          cd emsdk
+          ./emsdk install latest
```

Comment:
It's not as good an idea to cache git repos imo, also it's a very fast download. Instead of downloading from GitHub (repo) we'd be downloading from GitHub (actions cache). Not much of a difference :smile: